### PR TITLE
Access bank

### PR DESCRIPTION
**Pull request by saminume**
Updated Access bank logo and removed Diamond bank.
Issue #80

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (demilad) [inline .vscode/launch.json]: Nice! Can you please add this to `.gitignore` and also don't forget to optimise both SVG and PNG using the tools highlighted [here](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md).
Also there seem to be some merge conflicts with the changed files, please check that too

Comment (saminume) [reply .vscode/launch.json]: Thanks @demilad I'll just do that, the PNG and SVG are both optimized.

Comment (aeadedoyin) [reply .vscode/launch.json]: Hi @saminume 
This as already been done and merged. #81

Comment (saminume) [reply .vscode/launch.json]: Nice! @aeadedoyin

**Comment by aeadedoyin** [comment]
Thanks for your contribution!

**Pull request closed without merging**